src/pyscript/_generator.py

```diff
@@ -100,8 +100,12 @@ def create_project(
             assert app_or_file_name is not None
             app_name = app_or_file_name.removesuffix(".py")
     else:
-        assert app_or_file_name is not None
-        app_name = app_or_file_name
+        if app_or_file_name and app_or_file_name.endswith(".py"):
```

Comment:
out of curiosity, as you pass a default, in which case app_or_file_name is `None` ?

Reply:
We are only asking the user for a name if they don't pass a name and a command, and that way we will provide a default name. If they do something like `pyscript -c print("hello") --wrap` the name here would be None.

This is why typing is seeing app_or_file_name to maybe be None